Lib/test/pickletester.py

```diff
@@ -3499,6 +3499,37 @@ class MyClass:
                         ValueError, 'The reducer just failed'):
                     p.dump(h)
 
+    def test_reducer_override_no_reference_cycle(self):
```

Comment:
You should mark this test CPython-only, as it relies on reference counting. Use the `@support.cpython_only` decorator.